build/cluster-operator-ansible/playbooks/cluster-api-prep/deploy-cluster-api.yaml

```diff
@@ -29,6 +29,13 @@
     import_role:
       name: lib_openshift
 
+  - name: copy cluster-operator binary to master
+    copy:
+      src: files/aws-machine-controller
+      dest: /usr/bin/aws-machine-controller
+      setype: container_file_t
+      mode: 0755
```

Comment:
If you broke this out into a separate play run on hosts: masters, would we have multi-master support?